### DISCUSSION TITLE
Using Style="color:#778CA2" on ProperyColumn also changes the style of its header

### DISCUSSION THREAD

**Discussion by thieng101** [microsoft/fluentui#33514] [Q&A]
Hello, I followed the instructions on the demo website to change the color of the text in a PropertyColumn. It is supposed to have the column change while its header remains unchanged, but it actually changes both the header and column (as shown in the picture).

This is my code:
  `<PropertyColumn Property="c => c.id" Style="color: #af5f00 ;" Align="Align.Start" Title="Order ID"/>`
  
  
<img width="386" alt="image" src="https://github.com/user-attachments/assets/2bab99fb-d265-4660-ae8f-7e932cc6e274" />

Please help. I'd very appreciate.

**Comment by ValentinaKozlova** [comment]
Could you please provide a link on stackblitz  for your example? What demo site did you use (V9, V8, web components)?
